pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-osd/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, pidgin, xosd
+, autoconf, automake, libtool } :
+
+stdenv.mkDerivation rec {
+  name = "pidgin-osd-0.1.0";
+  src = fetchurl {
+    url = https://github.com/mbroemme/pidgin-osd/archive/pidgin-osd-0.1.0.tar.gz;
+    sha256 = "11hqfifhxa9gijbnp9kq85k37hvr36spdd79cj9bkkvw4kyrdp3j";
+  };
+
+  preConfigure = "./autogen.sh";
```

Comment:
Ooh, I figured there had to be something like that, but just couldn't find it.  Unfortunately, naively swapping that in gave me
```
Makefile.am: error: required file './ChangeLog' not found
```
I don't see any obvious references to ChangeLog in the project or in a couple minutes of looking at autoreconfHook.  I should have some time to dig in in the next few days.

(And for future reference, I'm guessing this should be a single commit to keep the nixpkgs history clean, and force-pushing tends to make the github conversation ugly, so I should open a new PR on a new branch with the new single commit?  Or is it okay to have two commits in the PR?)